packages/@aws-cdk/aws-redshift/lib/parameter-group.ts

```diff
@@ -53,6 +111,10 @@ export class ClusterParameterGroup extends ClusterParameterGroupBase {
   public static fromClusterParameterGroupName(scope: Construct, id: string, clusterParameterGroupName: string): IClusterParameterGroup {
     class Import extends Resource implements IClusterParameterGroup {
       public readonly clusterParameterGroupName = clusterParameterGroupName;
+      protected updatableParameters = false;
```

Comment:
I don't understand why `updatableParameters` exists. Why is it not enough to simply check if the `parameter` in question has already been defined?

Reply:
You're right. It doesn't need to exist, but similar to my comment above I wanted it to remain consistent with the `Cluster` implementation.

Ideally I would think that this method should look like this where I wouldn't need to implemented the method and just set 
`updateableParameters` as False

```ts
  public static fromClusterParameterGroupName(scope: Construct, id: string, clusterParameterGroupName: string): IClusterParameterGroup {
    class Import extends ClusterParameterGroupBase {
      public readonly clusterParameterGroupName = clusterParameterGroupName;
      protected updatableParameters = false;
    }
    return new Import(scope, id);
  }
 ```